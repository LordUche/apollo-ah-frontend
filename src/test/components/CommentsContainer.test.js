import '@babel/polyfill';
import React from 'react';
import setup from '../setup';
import ConnectedCommentsContainer from '../../components/CommentsContainer';

const container = <ConnectedCommentsContainer slug="abc" />;

describe('<CommentsContainer>', () => {
  it('should render without crashing', () => {
    const wrapper = setup(container);
    expect(wrapper).toMatchSnapshot();
  });
});
