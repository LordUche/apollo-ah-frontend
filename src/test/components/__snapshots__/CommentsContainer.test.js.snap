// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CommentsContainer> should render without crashing 1`] = `
<Provider
  store={
    Object {
      "clearActions": [Function],
      "dispatch": [Function],
      "getActions": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <BrowserRouter>
    <Router
      history={
        Object {
          "action": "POP",
          "block": [Function],
          "createHref": [Function],
          "go": [Function],
          "goBack": [Function],
          "goForward": [Function],
          "length": 1,
          "listen": [Function],
          "location": Object {
            "hash": "",
            "pathname": "/",
            "search": "",
            "state": undefined,
          },
          "push": [Function],
          "replace": [Function],
        }
      }
    >
      <Connect(CommentsContainer)
        slug="abc"
      >
        <CommentsContainer
          clearComments={[Function]}
          commentMessage="Done"
          fetchComment={[Function]}
          gettingComments={false}
          hasMoreComments={false}
          isLoggedIn={false}
          newCommentList={
            Object {
              "authorImage": "image.jpg",
              "authorName": "John",
              "body": "Hello world",
              "date": "July 4th 2018",
              "id": 1,
            }
          }
          oldCommentList={Array []}
          postDislikeComment={[Function]}
          postLikeComment={[Function]}
          postingComment={false}
          profile={
            Object {
              "errorData": Array [],
              "firstname": "John",
              "image": "image.jpg",
              "username": "johnny",
            }
          }
          remainingComments={0}
          sendComment={[Function]}
          slug="abc"
          token=""
        >
          <div
            className="comment-grp card"
          />
          <PostComment
            body=""
            commentMessagef="Done"
            handleInputChange={[Function]}
            handlePostComment={[Function]}
            isLoggedIn={false}
            postingComment={false}
          >
            <div
              className="comment-grp card mt-3 relative"
            >
              <form
                className="form"
                onSubmit={[Function]}
              >
                <div
                  className="form-group"
                >
                  <input
                    autoComplete="off"
                    className="form-control"
                    disabled={false}
                    name="body"
                    onChange={[Function]}
                    placeholder="Leave a comment"
                    value=""
                  />
                  <button
                    className="btn btn-link"
                    disabled={false}
                    type="submit"
                  >
                    <i
                      className="fas fa-paper-plane"
                    />
                    <span
                      className="spinner-grow text-dark"
                    />
                  </button>
                </div>
              </form>
              <Link
                className="link-surface"
                replace={false}
                to="/login"
              >
                <a
                  className="link-surface"
                  href="/login"
                  onClick={[Function]}
                />
              </Link>
            </div>
          </PostComment>
        </CommentsContainer>
      </Connect(CommentsContainer)>
    </Router>
  </BrowserRouter>
</Provider>
`;
